server/protocols/webadmin/webadmin-jmap/src/test/java/org/apache/james/webadmin/data/jmap/RunRulesOnMailboxRoutesTest.java

```diff
@@ -568,12 +563,12 @@ void runRulesOnMailboxShouldApplyFlagCriteria() throws Exception {
                   {
                     "comparator": "isSet",
                     "field": "flag",
-                    "value": "\\Seen"
+                    "value": "Seen"
```

Comment:
There seems to be some json deserialization issue with the `\\` at the moment so for now I took it out. Will check it out later. 

Or maybe we can just omit in the rule the `\`?

Reply:
Regarding this... In the test I need to put `"value": "\\\\Seen" ` for it to work, because while debugging I could observe that sparks does the escape when we get the request body, and then json deserializer is doing it too behind. 

Can we just do without the `\` for the sake of simplicity?